custom-war-packager-lib/src/main/java/io/jenkins/tools/warpackager/lib/config/DockerBuildSettings.java

```diff
@@ -0,0 +1,60 @@
+package io.jenkins.tools.warpackager.lib.config;
+
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
+
+/**
+ * Settings for Docker packaging
+ * @author Oleg Nenashev
+ */
+public class DockerBuildSettings {
+
+    public static final String DEFAULT_BASE = "jenkins/jenkins:lts";
```

Comment:
I thought it was `jenkinsci/jenkins`?

Reply:
Also can we pick up the tag based on the Jenkins core version detected somewhere? Or does it not really matter anyway?

Reply:
I can expose it to CLI parameter. Created https://issues.jenkins-ci.org/browse/JENKINS-51333 so that I follow-up on it later. It would be useful of-course

Reply:
> I thought it was jenkinsci/jenkins?

Yes